main/tinc/APKBUILD

```diff
@@ -47,7 +47,7 @@ package() {
 	cd "$_builddir"
 	make DESTDIR="$pkgdir" install || return 1
 
-	mkdir "$pkgdir"/etc/tinc
+	mkdir -p "$pkgdir"/etc/tinc
```

Comment:
This `mkdir` is not be needed at all, `install -D` automatically creates intermediate repositories.